<ui version="4.0" >
 <class>ViewerPresetsWidget</class>
 <widget class="QWidget" name="ViewerPresetsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QListWidget" name="m_presetsList" />
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2" >
     <item>
      <widget class="QToolButton" name="m_addButton" >
       <property name="toolTip" >
        <string>add</string>
       </property>
       <property name="text" >
        <string>+</string>
       </property>
       <property name="icon" >
        <iconset resource="../qosmic.qrc" >
         <normaloff>:/icons/silk/bullet_add.xpm</normaloff>:/icons/silk/bullet_add.xpm</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_delButton" >
       <property name="toolTip" >
        <string>remove</string>
       </property>
       <property name="text" >
        <string>-</string>
       </property>
       <property name="icon" >
        <iconset resource="../qosmic.qrc" >
         <normaloff>:/icons/silk/bullet_delete.xpm</normaloff>:/icons/silk/bullet_delete.xpm</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_updateButton" >
       <property name="toolTip" >
        <string>update preset</string>
       </property>
       <property name="text" >
        <string>&lt;></string>
       </property>
       <property name="icon" >
        <iconset resource="../qosmic.qrc" >
         <normaloff>:/icons/silk/bullet_wrench.xpm</normaloff>:/icons/silk/bullet_wrench.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_upButton" >
       <property name="toolTip" >
        <string>move up</string>
       </property>
       <property name="icon" >
        <iconset resource="../qosmic.qrc" >
         <normaloff>:/icons/silk/bullet_arrow_up.xpm</normaloff>:/icons/silk/bullet_arrow_up.xpm</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="m_downButton" >
       <property name="toolTip" >
        <string>move down</string>
       </property>
       <property name="icon" >
        <iconset resource="../qosmic.qrc" >
         <normaloff>:/icons/silk/bullet_arrow_down.xpm</normaloff>:/icons/silk/bullet_arrow_down.xpm</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qosmic.qrc" />
 </resources>
 <connections/>
</ui>
